glide.lock

```diff
@@ -9,28 +9,99 @@ imports:
   version: 4c0e84591b9aa9e6dcfdf3e020114cd81f89d5f9
   subpackages:
   - quantile
+- name: github.com/boltdb/bolt
```

Comment:
Fortunately it's not much bigger ~ OSX statsdex build is 28MB and Linux statsdex build is 26MB. 

I really wanted to start putting test imports into the appropriate place in Glide but ran into one [issue] so avoided going down that rabbit hole. 

[issue]: https://github.com/Masterminds/glide/issues/597